.github/workflows/test.yml

```diff
@@ -1,6 +1,13 @@
 name: tests
 
-on: [push, pull_request]
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
+  release:
+    types:
+      [published]
```

Comment:
It's nice to have `workflow_dispatch` as well to be able start tests when necessary.

Reply:
added